### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by H-Yulong**
User authentication service and UI.
You can now login/logout smoothly.

### CODE REVIEW

**Comment by calintat** [comment]
This looks quite good, thanks Yulong!

I think we don't need the top button for logging out, it's fine to just have it as a normal button (like you already do).

Also I'm not sure if we should have yet another drawer for account management. I think it's fine to just put this in the "Project" drawer, either above or below the project metadata and import/export buttons. Maybe with a line separating the project and account sections.

**Comment by jamievicary** [comment]
Hi Calin, we decided to do it like this during the standup last week. There is a lot to go in there potentially.

**Comment by calintat** [comment]
Ok that's fine, I just think we're getting to the point where we have too many drawers.

File: Makefile.toml [hunk 1]
```diff
@@ -79,4 +79,4 @@ args = [
 ]
 
 [tasks.serve]
-run_task = { name = ["build-highs", "dist-static-watch", "build-wasm-watch", "devserver"], fork = true, parallel = true }
+run_task = { name = ["dist-static-watch", "build-wasm-watch", "devserver"], fork = true, parallel = true }
```
Comment (regular-citizen) [inline Makefile.toml:82]: I'm curious about this change. Is the highs build broken?

Comment (H-Yulong) [reply Makefile.toml:82]: Nah, I just disabled it because I'm not using highs and it is taking my VM's disk space. Will add it back before merging.

Comment (regular-citizen) [reply Makefile.toml:82]: Cool, makes sense.

**Comment by calintat** [comment]
@H-Yulong I have a small request: could you please swap the Account and Project buttons? It makes more sense if the ordering is Account > Project > Signature as that's the correct hierarchy.

**Comment by calintat** [comment]
> I think we don't need the top button for logging out, it's fine to just have it as a normal button (like you already do).

And feel free to ignore this, I changed my mind.

**Pull request merged**